folly/lang/Align.h

```diff
@@ -120,7 +120,7 @@ struct alignas(max_align_v) max_align_t {};
 //
 //  mimic: std::hardware_destructive_interference_size, C++17
 constexpr std::size_t hardware_destructive_interference_size =
-    kIsArchArm ? 64 : 128;
+    kIsArchArm | kIsArchS390 ? 64 : 128;
```

Comment:
This should be boolean or `||` and not bitwise or `|` and I would put the condition in parens, so `(kIsArchArm || kIsArchS390)`